### DISCUSSION TITLE
Any hope for V2?

### DISCUSSION THREAD

**Discussion by phantum29** [gtxaspec/wz_mini_hacks#723] [Q&A]
As others have mentioned, there are issues with the new 'secure' (aka bare minimum security practices as wyze were caught with their poor practices) and I can't seem to compile any firmware to work with the v2 beyond setup. I suspect the kernel provided in the v2_install won't work as they might have updated that with their upgrade. Attempting to use more 'modern' firmware after modifying compile_image.sh to recognize the past 5 versions released still results in 'connection timed out' when connecting during setup. Maybe there's some verification of the firmware and/or phoning home, as, during wifi setup, it does connect to my wifi, responds to pings for about a minute, then disconnects and reports 'connection timed out.' I use wz_mini_hacks for my v3s and found some uses for my old v2 and would love to get them to work with wz_mini rather than using other solutions.

**Comment by bjs-pdx** [comment]
You might want to use an earlier version of the firmware:

**Latest Firmware for V2 and Pan: Firmware versions beyond 4.9.8.1002/4.10.8.1002 are compatible but do not support audio via RTSP streaming. Resolution for this issue is pending in fixes #347.**
